modules/util/keepRight/errorSchema.json

```diff
@@ -217,7 +217,7 @@
             },
             "_220": {
                 "title": "misspelled tags",
-                "description": "This (node|way|relation) is tagged '([\\w:]+)=(.+)' where &quot;(\\2|\\3)&quot; looks like &quot;([\\w\\s]+)&quot;",
+                "description": "This (node|way|relation) is tagged '([\\w]+)(:([\\w]+))?=(.+)' where &quot;(\\2|\\3|\\4|\\5)&quot; looks like &quot;([\\w\\s]+)&quot;",
```

Comment:
Hmm, this is an interesting case. I have a feeling this won't work for all cases because the pattern will now only match keys with 0 or 1 namespace (which doesn't handle the likes of `turn:lanes:backward`).

I see that the bit in quotes can be any substring of the key though (I wonder if that also applied to the value). It might be easiest to revert the key group and then just use a `.+` in the group within the quotes.